dataland_analytics/analysis_scripts/scripts_with_api_admin/opened_and_closed_requests.py

```diff
@@ -0,0 +1,42 @@
+import dataland_requests
+from dataland_analytics.config import *
+from dataland_requests.models.request_status import RequestStatus
+
+OBSERVATION_PERIOD_START_EPOCH = 1714588322000
```

Comment:
Maybe a comment with a human-readable time would be helpful

Reply:
another option would be to compute it on the fly like `np.datetime64("2024-05-01T18:32", "ms").astype('int64')`

Reply:
As epoch fits the dataland data model this is imo good enough.